src/views/v2/widgets/events-list/event/date.php

```diff
@@ -0,0 +1,29 @@
+<?php
+/**
+ * Widget: Events List Event Date
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/events/v2/widgets/events-list/event/date.php
+ *
+ * See more documentation about our views templating system.
+ *
+ * @link http://m.tri.be/1aiy
+ *
+ * @version TBD
+ *
+ * @var WP_Post $event The event post object with properties added by the `tribe_get_event` function.
+ *
+ * @see tribe_get_event() For the format of the event object.
+ */
+use Tribe__Date_Utils as Dates;
+
+$event_date_attr = $event->dates->start->format( Dates::DBDATEFORMAT );
+
+?>
+<div class="tribe-events-widget-events-list__event-datetime-wrapper tribe-common-b2 tribe-common-b3--min-medium">
+	<?php $this->template( 'widgets/events-list/event/date/featured', [ 'event' => $event ] ); ?>
+	<time class="tribe-events-widget-events-list__event-datetime" datetime="<?php echo esc_attr( $event_date_attr ); ?>">
+		<?php echo $event->schedule_details->value(); ?>
```

Comment:
Drop a note here to indicate this is not escaped with the knowledge that what we get back is HTML built by us and already escaped.

Reply:
Done.